crates/oxc_linter/src/snapshots/eslint_vars_on_top.snap

```diff
@@ -0,0 +1,205 @@
+---
+source: crates/oxc_linter/src/tester.rs
+snapshot_kind: text
+---
+  ⚠ eslint(vars-on-top): All 'var' declarations must be at the top of the function scope.
+   ╭─[vars_on_top.tsx:6:4]
+ 5 │             }
+ 6 │             var second = 0;
+   ·             ───────────────
+   ╰────
+
+  ⚠ eslint(vars-on-top): All 'var' declarations must be at the top of the function scope.
+   ╭─[vars_on_top.tsx:7:7]
+ 6 │                first = 4;
+ 7 │                var second = 1;
+   ·                ───────────────
+ 8 │                second = 2;
+   ╰────
+
```

Comment:
lets add a help.

"Consider moving this to the top of the functions scope or using let or const to declare this variable"

Reply:
added: https://github.com/oxc-project/oxc/pull/8157/commits/a1dfdf607abded6ece0e45698269e03490403678